scripts/osm_correction.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+UPDATE planet_osm_point SET name ='Association des Musulmans de Fribourg'  WHERE osm_id = 1351136417;
```

Comment:
does it somthing we can plan with code? Or you mean when we upgrade the demo app?